models/demos/llama3/lt

```diff
@@ -46,12 +46,39 @@ def ensure_ttsmi_installed():
     config_file = os.path.join(config_dir, f"reset_config_{hostname}.json")
 
     try:
-        subprocess.run(["tt-smi", "-g", config_file], check=True)
+        if "glx" in hostname:
+            generate_tg_config_file()
+        else:
+            subprocess.run(["tt-smi", "-g", config_file], check=True)
         print(f"Generated reset config file: {config_file}")
     except subprocess.CalledProcessError as e:
         print(f"Error generating reset config file: {e}")
         sys.exit(1)
 
+    print("Resetting device on program start...")
+    reset_device_sync(config_file)
```

Comment:
@johanna-rock-tt

Reply:
For TG I got regular hangs at the very start, so the first test just hangs on init devices then. Resetting at start fixes that.

Reply:
There was actually also an issue with cancelling a job where I believe we didn't reset properly and then the next job hangs in init. But I removed this after fixing the code after rebasing because a lot of the code had changed and I didn't run any extended tests after that. Not sure if that's only an issue for TG or in the general case.

Reply:
I see. We can keep this then 👍 
It's a minor inconvenience for extra reliability.

Do you know if these hangs, assuming they were caused by a device in a bad state, would be visible on tt-smi? If so we can probably modify the code to check tt-smi first and if the device is in bad state, reset at start. 

No strong feeling for this method though, more curious than anything.

Reply:
I don't know if we see it in tt-smi, we could check and change it, yeah. The hang was definitely because of a bad device state since it hung in the init devices already (no model code ru
n at that point).